tests/Tree.spec.tsx

```diff
@@ -1149,24 +1149,6 @@ describe('Tree Basic', () => {
     ).toHaveLength(2);
   });
 
-  it('support virtual', () => {
```

Comment:
Why delete this case?

Reply:
I moved the test to TreeVirtual.spec.tsx. The test worked in this file even if we set the virtual parameter to true. It is because virtual list is mocked, and it is always with virtual=false (https://github.com/react-component/virtual-list/blob/master/src/mock.tsx)